CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Stores/Account/AccountStore.cs

```diff
@@ -110,5 +110,26 @@ private async Task<AccountDto> FetchDriverDataAsync(int accountId)
 
             return result;
         }
+
+        public async Task<AccountDto> UpdateAccountAsync(AccountDto account)
+        {
+            if (account == null)
+            {
+                throw new ArgumentNullException(nameof(account), "Account cannot be null.");
+            }
+
+            var json = JsonConvert.SerializeObject(account);
+            var response = await _client.PutAsync($"accounts/{account.Id}", json);
+
+            response.EnsureSuccessStatusCode();
+
+            var responseBody = await response.Content.ReadAsStringAsync();
+
+            var updatedAccount = JsonConvert.DeserializeObject<AccountDto>(responseBody);
```

Comment:
if API client will work with generic types we will be able to replace all these lines with this.

`var response = await _client.PutAsync<AccountDto, AccountDto>($"accounts/{account.Id}", account);`